### ISSUE TITLE
Polygon2D opacity doesn't work.

### ISSUE THREAD

**Issue by sergicollado** [godotengine/godot#5151]
commit 00bfdc834bc028e7c15e939662a72353250d00f3
OpenSuse 42.1 KDE x86_64    Intel Core i5-4670 CPU @ 3.40GHz

In Polygon2D node opacity parameter doesn't work.

**Comment by rgrams** [comment]
+1. Same on Windows 7, various builds.

**Comment by reduz** [comment]
aware of this, but to fix requires many hacks to a renderer that will be rewritten after 2.1 is out, so leaving this for then

**Comment by Zylann** [comment]
Argh I'll have to drop all my polygons now and use sprites :p

**Comment by rgrams** [comment]
You can easily get around this bug with a shader (though you won't get _quite_ the same ease of use). Just do: 

`uniform float opacity = 1.0;`
`COLOR.a *= opacity;`

If you're not using any other shaders you can put it on any node and set it's children to "Use Parent" material so you can control multiple nodes at once, like regular opacity.

**Comment by akien-mga** [comment]
See also #5544 and #6060 which are duplicates of this one. #6060 seems to infer that this might be a regression in 2.1.

**Comment by ArcFutahito** [comment]
Encountered same problem trying to make some UI elements. Now I'm REALLY waiting on 3.0

**Comment by bojidar-bg** [comment]
Whats the status of this in current master (assuming polygon2d still works...)?

**Comment by Zylann** [comment]
Not sure if we can even see it given this https://github.com/godotengine/godot/issues/7840

**Comment by reduz** [comment]
this was fixed in 3.0 and #7840 solved, so closing this
